salt/ufw/lib_ufw/user.rules

```diff
@@ -29,6 +29,9 @@
 ### tuple ### allow tcp 22 0.0.0.0/0 any 0.0.0.0/0 in_tun0
 -A ufw-user-input -i tun0 -p tcp --dport 22 -j ACCEPT
 
+### tuple ### allow tcp 64378 0.0.0.0/0 any 0.0.0.0/0 in_tun0
+-A ufw-before-input -i tun0 -p udp --sport 64738 -j ACCEPT
```

Comment:
Code and comment do not seem to agree - comment says `tcp` and code `udp`. Is this expected? Why is this rule needed and what is its effect in detail?